commands/evm-call.js

```diff
@@ -0,0 +1,55 @@
+const exitOnError = require('../utils/exit-on-error');
+const web3 = require('web3');
+const { NearProvider, utils } = require('near-web3-provider');
+const assert = require('assert');
+
+module.exports = {
+    command: 'evm-call <evmAccount> <contractName> <methodName> [args]',
```

Comment:
Let's replace `<evmAccount>` with `<nearEvmAccountId>`. Contract name with `<evmContractName>` to avoid confusion between names

Reply:
replaced